rfcs/044-sierra-series/README.md

```diff
@@ -0,0 +1,503 @@
+
+*   Some series have thousands of items, e.g. "ACLS Humanities E-Book." appears in 5k+ instances of 830 Series Added Entry-Uniform Title.
+
+*   "Early European Books : Printed sources to 1700" appears in 31k+ instances of 440 Series Statement, divided into individual volumes.
+
+*   We should consider dismissing or removing certain values in subfield $a as unlikely to be useful for navigation, including:
+
+    *   ` ` (a single space) which appears in 1k+ instances of 830
+
+
+
+## How we will model these relationships in the catalogue API
+
+Previously, all the relationships in the catalogue API have been to other Works.
+We're not going to create Works for series (unless the Library team have created one in Sierra).
+
+We will use use the `parts` and `partOf` properties on the Work model, that are current used to represent archive collections.
+
+This is how relations are modelled for the archive collection on [dq3spb42](https://api.wellcomecollection.org/catalogue/v2/works/dq3spb42?include=parts,partOf):
+
+```json
+"parts": [
+  {
+    "id": "fncn55x6",
+    "title": "Articles on Nutrition Education",
+    "referenceNumber": "PP/DSM/3/1",
+    "totalParts": 0,
+    "totalDescendentParts": 0,
+    "type": "Work"
+  },
+  ...
+  {
+    "id": "k4vp98bp",
+    "title": "Miscellaneous Items",
+    "referenceNumber": "PP/DSM/3/6",
+    "totalParts": 0,
+    "totalDescendentParts": 0,
+    "type": "Work"
+  }
+],
+"partOf": [
+  {
+    "id": "hb9hqd8p",
+    "title": "McLaren, Dr Donald S",
+    "referenceNumber": "PP/DSM",
+    "partOf": [],
+    "totalParts": 3,
+    "totalDescendentParts": 9,
+    "type": "Collection"
+  }
+],
+```
+
+These are [rendered on the website](https://wellcomecollection.org/works/dq3spb42) as a collapsible hierarchy:
+
+![](archives.png)
+
+Some series can have thousands of entries, so this UI isn't practical – we'll need to render the information differently.
+We'll determine how to render them based on the `type` property of the entry in `partOf`.
+
+*   For archive works, we'll use `partOf: Collection` and render an archive tree
+*   For works which are part of a series (440, 490, 830), we'll use `partOf: Series` and display a link as part of the work metadata (similar to Encore)
+*   For works which have constituent parts or are part of something else, we'll use `partOf: Work`
+
+If a series or host item entry has a volume, we'll use a nested `partOf` property to record the series and the volume individually (see example below).
+
+If a series or host item entry has an identifier (such as an ISSN), we'll add it to the `partOf` and mint a canonical identifier for it.
+
+### Examples
+
+-   b10747850
+
+    ```
+    490 1  Morphogenesis of the vertebrate brain ;|v2
+    830  0 Morphogenesis of the vertebrate brain ;|v2
+    ```
+
+    We'll deduplicate because the information in 490 and 830 is the same:
+
+    ```json
+    "partOf": [
+      {
+        "title": "Morphogenesis of the vertebrate brain; 2",
```

Comment:
If the relationship could be presented in a flat object, then I think that maintaining both the $a and $v or $g subfields in this property is probably redundant.
e.g. 
`{"subpart": "2", "title": "Morphogenesis of the vertebrate brain"}`, rather than repeating the title of the superobject in both properties